src/TitleDetails/PiecesList/PiecesList.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { MultiColumnList } from '@folio/stripes/components';
+import { FolioFormattedDate } from '@folio/stripes-acq-components';
+import { PIECE_STATUS } from '../constants';
+
+const columnMapping = {
+  title: <FormattedMessage id="ui-receiving.piece.title" />,
+  caption: <FormattedMessage id="ui-receiving.piece.caption" />,
+  format: <FormattedMessage id="ui-receiving.piece.format" />,
+  receiptDate: <FormattedMessage id="ui-receiving.piece.receiptDate" />,
+  receivedDate: <FormattedMessage id="ui-receiving.piece.receivedDate" />,
+};
+const VISIBLE_COLUMNS = ['title', 'caption', 'format'];
+
+const PiecesList = ({ pieces, title, pieceStatus }) => {
```

Comment:
IMHO, `pieceStatus` is hard-to-extend solution. I see `ExpectedPiecesList` and `ReceivedPiecesList` components, that pass custom props to the common `PiecesList`